editor/src/components/shared/project-components.ts

```diff
@@ -502,6 +512,36 @@ export const fragmentComponentInfo: ComponentInfo = {
   },
 }
 
+export const mapComponentInfo: ComponentInfo = {
+  insertMenuLabel: 'List',
+  elementToInsert: (): JSXMapExpressionWithoutUID => ({
+    type: 'JSX_MAP_EXPRESSION',
+    javascriptWithUIDs: '[1, 2, 3].map((v) => <\nPlaceholder data-uid="placeholder-id" />);',
```

Comment:
did you hand write this? IIRC if you don't wrap <Placeholder in parenthesis, it's not valid jsx

Reply:
I copied this from the `projectContents` of a project